scripts/host.sh

```diff
@@ -43,13 +44,24 @@ download_dependencies() {
     msg "downloading axelard binary $axelard_binary"
     if [[ ! -f "${axelard_binary_path}" ]]; then
         local axelard_binary_url
-        
+        local axelard_binary_signature_url
+
         if [ "$network" == "hacknet" ]; then
             axelard_binary_url="https://axelar-hackathons.s3.us-east-2.amazonaws.com/avalanche-summit/binaries/${axelard_binary}"
         else
             axelard_binary_url="https://axelar-releases.s3.us-east-2.amazonaws.com/axelard/${axelar_core_version}/${axelard_binary}"
+            axelard_binary_signature_url="https://axelar-releases.s3.us-east-2.amazonaws.com/axelard/${axelar_core_version}/${axelard_binary}.asc"
         fi
         curl -s --fail "${axelard_binary_url}" -o "${axelard_binary_path}" && chmod +x "${axelard_binary_path}"
+        if [ -n "$axelard_binary_signature_url" ]; then
+          curl -s --fail "${axelard_binary_signature_url}" -o "${axelard_binary_signature_path}"
+          curl https://keybase.io/axelardev/key.asc | gpg --import
+          printf "\nVerifying Signature of binary. Output: \n================================================"
+          gpg --verify "${axelard_binary_signature_path}" "${axelard_binary_path}"
```

Comment:
Is gpg installed by default? We should skip it otherwise and tell validators to install it

Reply:
it is on linux and macs.